### PR TITLE
Julian/nav bar

### PR DESCRIPTION
**Pull request by EzyhB**

### CODE REVIEW

File: src/components/Nav/index.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Drawer, AppBar, Box, Toolbar, IconButton, Typography, Container, Avatar, Tooltip, Button } from '@mui/material';
+import TextField from '@mui/material/TextField';
+import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
+import logoLightMode from '../../assets/images/fit_trax_app_logo_lightmode.png';
+import { avatarStyle, drawerStyle, iconButtonStyle, navStyle, logoStyle, boxStyle, buttonStyle, drawerPaperStyle } from './styles';
+
+const Nav = () => {
+  const [openDrawer, setOpenDrawer] = React.useState(false);
+  const [username, setUsername] = React.useState('');
```
Comment (EzyhB) [inline src/components/Nav/index.tsx:10]: do these need an initial state thats not an empty string?

File: src/components/Nav/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      return;
+    }
+    setOpenDrawer(open);
+  };
+
+  const handleFormSubmit = () => {
+    /**
+     * @TODO: Waiting for API to submit the form
+     */
+    console.log('form gender :>> ', gender);
```
Comment (EzyhB) [inline src/components/Nav/index.tsx:26]: do we still need the console logs?

File: src/components/Nav/index.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    console.log('form username :>> ', username);
+    setOpenDrawer(false);
+  };
+
+  const handleFormCancel = () => {
+    setOpenDrawer(false);
+  };
+
+  return (
+    <AppBar sx={navStyle} position="static">
```
Comment (EzyhB) [inline src/components/Nav/index.tsx:38]: does this need to be wrapped inside of another container?

File: src/components/Nav/index.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+              sx: drawerPaperStyle,
+            }}
+            anchor="right"
+            open={openDrawer}
+            onClose={toggleDrawer(false)}
+          >
+            <Box component="form" sx={drawerStyle} noValidate autoComplete="off">
+              <h3>User Account</h3>
+
+              <Box sx={boxStyle}>
```
Comment (EzyhB) [inline src/components/Nav/index.tsx:66]: you can try mapping an array for each one of these boxes rather than hard coding them all in?

Comment (Jul-Me) [reply src/components/Nav/index.tsx:66]: Sure, will do the refactoring when I add the API integration. This is a v1.

File: src/components/Nav/index.tsx (hunk 2 above)
Comment (Jul-Me) [reply src/components/Nav/index.tsx:26]: The console logs are there to show the local state till we can intergrade with the aPI and save the data. I also added a TODO comment for better understanding. Happy to remove the console logs though if they cause major issues.

File: src/components/Nav/index.tsx (hunk 1 above)
Comment (Jul-Me) [reply src/components/Nav/index.tsx:10]: No they need to be empty for now, until we intergrade with the API and populate the data.

**Pull request merged**